js/gdm/authPrompt.js

```diff
@@ -633,10 +657,10 @@ var AuthPrompt = class {
         // It must be shortened after the Endless 3.2 release, once there is
         // time to translate the shortened version.
         // Translators: During a password reset, prompt for the "secret code" provided by customer support.
-        this.setQuestion(_("Enter unlock code provided by customer support:"));
+        this.setQuestion(_("Enter unlock code:"));
```

Comment:
You can delete the FIXME above, having made this change.

Reply:
Of course, sorry.